components/expenses/Transaction.js

```diff
@@ -152,6 +171,22 @@ class Transaction extends React.Component {
             {this.renderPaymentOrigin()}
             {' | '}
             <Moment relative={dateDisplayType === 'interval'} value={createdAt} />
+            {expense && (
+              <Fragment>
+                {' | '}
+                <Link
+                  route="expenses"
+                  params={{
+                    collectiveSlug: collective.slug,
+                    filter: 'categories',
+                    value: expense.category,
+                  }}
+                  scroll={false}
```

Comment:
What is this param for?

Reply:
@Betree I copied the component from `components/expenses/expense.js` 
```
<Link
                  route="expenses"
                  params={{
                    collectiveSlug: expense.collective.slug,
                    filter: 'categories',
                    value: expense.category,
                  }}
                  scroll={false}
                >
                  {capitalize(expense.category)}
                </Link>
```

Reply:
@Betree should I remove it in my next commit?

Reply:
From https://nextjs.org/docs#disabling-the-scroll-changes-to-top-on-page I understand that this is to avoid scrolling on top of the page when link changes. I'd say that we do want to scroll back on top for this one, so we should probably remove the prop yes.